### ISSUE TITLE
Cropping does not work if the user does not change the default bounding box (React component)

### ISSUE THREAD

**Issue by sturatcliffe** [scaleflex/filerobot-image-editor#298]
Hey,

I have an instance of the React component configured so that the default (and only) tool available is the crop tool, so when the component first loads it shows the default crop boundary based on the aspect ratio provided.

However, if the user does not interact with the default boundary, the crop does not work and the output is the full image. If it matters, I'm not using the save modal, I'm using the ```getCurrentImgDataFnRef``` prop and extracting the base64 image data like so:

```
const {
   imageData: { imageBase64 },
} = getCurrentImgDataFnRef.current()
```

Thanks

**Comment by AhmeeedMostafa** [comment]
Could u please provide a codesandbox that contains ur code exactly to check it and have it reproduced?

**Comment by AhmeeedMostafa** [comment]
Okay we will check it once we have time and if it's found we will consider its fix and notify you

**Comment by AhmeeedMostafa** [comment]
it's fixed in v4.5.0 now, thanks for reporting.

**Comment by mkozen** [comment]
I have the same error in v4.5.1
